wcfsetup/install/files/lib/system/package/PackageArchive.class.php

```diff
@@ -426,13 +429,21 @@ protected function filterUpdateInstructions() {
 	 * @return	string		path to the dowloaded file
 	 */
 	public function downloadArchive() {
-		$prefix = 'package';
+		$tmpFile = FileUtil::getTemporaryFilename('package_');
```

Comment:
Should you delete this file in an error case?

Reply:
No, I would not like to change that in the pull-request, but only with a new one. Up to now the file was not deleted either (possibly also for debugging reasons).